src/NextGen/DonationForm/Controllers/DonationFormViewController.php

```diff
@@ -147,4 +154,25 @@ private function enqueueFormScripts(int $formId, string $formDesignId)
             'give'
         ))->loadInFooter()->enqueue();
     }
+
+    /**
+     * @unreleased
+     * @return void
+     */
+    private function enqueueFormStyles(DonationFormViewModel $viewModel)
+    {
+        wp_enqueue_global_styles();
+
+        // TODO: update font loading to be form design specific. Currently this is only used for classic form design until we introduce font settings to our form builder.
+        if ($viewModel->primaryFont() && $viewModel->designId() === ClassicFormDesign::id()) {
+            wp_enqueue_style(
+                'givewp-google-font',
+                'https://fonts.googleapis.com/css?family=' . urlencode(
+                    $viewModel->primaryFont()
+                ) . ':400,500,600,700&display=swap',
+                [],
+                GIVE_VERSION
+            );
+        }
```

Comment:
@jonwaldstein If we're moving away from the font setting for now, can we not have this hard-coded here? Could the Form Design enqueue it itself somehow?

Reply:
I removed the primary font stuff 3d3ac9aa

From our custom CSS conversation, there's nothing stopping a form design to add this to their stylesheet: 
```
@import url('https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&family=Montserrat:wght@300;400;500;700&display=swap');

body {
   font-family: 'Montserrat';
}
```

For now i'm just setting the font to `system-ui` which can easily be updated in the future using the above route or another approach.